roles/container_runtime/defaults/main.yml

```diff
@@ -137,6 +137,14 @@ l_crio_image_tag: "{{ l_crt_crio_image_tag_dict[openshift_deployment_type] }}"
 l_crio_image_default: "{{ l_crio_image_name }}:{{ l_crio_image_tag }}"
 l_crio_image: "{{ openshift_crio_systemcontainer_image_override | default(l_crio_image_default) }}"
 
+l_crio_pause_images_dict:
+  origin: 'docker.io/openshift/origin-${component}:${version}'
+  openshift-enterprise: 'registry.access.redhat.com/openshift3/ose-${component}:${version}'
+l_pause_registry_url_default: "{{ l_crio_pause_images_dict[openshift_deployment_type] }}"
+l_os_registry_url: "{{ oreg_url_master | default(oreg_url) | default(l_pause_registry_url_default) | regex_replace('${version}' | regex_escape, l_openshift_image_tag | default('${version}')) }}"
```

Comment:
I'm not sure ``some_var | default(foo) | default(bar) ...`` works.  Unless you know otherwise, I've always done it this way: ``some_var | default(foo | default(bar)) ...``